packages/server/controllers/guilds.js

```diff
@@ -14,7 +14,17 @@ async function getGuild(guildId) {
   return query;
 }
 
+async function getGuildUsers(guildId) {
+  const query = await prisma.guildMembers.findMany({
+    where: {
+      guildId: guildId,
+    },
+  });
+  return query;
```

Comment:
Right now, this query returns the user UUID and guild UUID for all the members of a guild. Since frontend wants to display members in a list, we want the Prisma query to fetch
- user's UUID
- user's first name
- user's last name

Your original idea for the Prisma query was more correct than what I said to change it to so my bad on that. After looking into it more [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#explicit-many-to-many-relations), we want to query the users table passing the guild UUID but it's different from a regular query. It will look something like this. 

![image](https://github.com/cppsea/icebreak/assets/36950420/71b32882-1eb0-4398-b3ec-6ae8d61c0abb)